Library/Formula/libmill.rb

```diff
@@ -0,0 +1,45 @@
+class Libmill < Formula
+  desc "Go-style concurrency in C"
+  homepage "http://libmill.org/"
+  url "http://libmill.org/libmill-0.16-beta.tar.gz"
```

Comment:
Not possible. If you look at http://libmill.org/download.html you would notice that there are no '-rc' or '-beta' free releases. That is because the project is still in version 0.x releases, which by many interpretations of semantic versioning means things are subject to change.